deeppavlov/models/tokenizers/spacy_tokenizer.py

```diff
@@ -17,6 +17,7 @@
 
 import spacy
 import spacy.language
+from nltk.corpus import stopwords
 
```

Comment:
consider testing the following code instead of nltk stopwords
and in case of using SpaCy we can omit the lang parameter

import spacy
sp = spacy.load('en_core_web_sm')
all_stopwords = sp.Defaults.stop_words